python/tidy/tidy.py

```diff
@@ -42,8 +43,33 @@
 sys.path.append(os.path.join(WPT_PATH, "tests"))
 sys.path.append(os.path.join(WPT_PATH, "tests", "tools", "wptrunner"))
 
-# Default configs
-config = {
+TCF = TypeVar("TCF")
+
+GenericCheckingFunction = Callable[[str, TCF], Iterator[tuple[int, str]]]
```

Comment:
I think it is more readable to have different concrete type aliases for the line and file checking functions, rather than having a single generic type.